addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -113,7 +113,7 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             }
         });
 
-        await browser.navigator.clipboard.writeText(this.target.querySelector('.o_survey_session_copy_url').textContent);
+        await browser.navigator.clipboard.writeText(this.target.querySelector('.o_survey_session_copy_url').value);
```

Comment:
Hello @adsh-odoo :slightly_smiling_face: 

Actually there are 2 elements with the `o_survey_session_copy_url` class in the template.
One is a `span` to copy the link at the start of the survey, the other one is an `input` to copy the link during the survey. So modifying the `textContent` to `value` isn't a solution as it will in turn make the one from the starting page fail.

What is probably best to do is replacing the `input` element with a `span` so that both elements with the class are spans and keep the js unchanged. However, as it is a fix in stable we need to avoid modifying the xml structure, so we could just move the `o_survey_session_copy_url` class to this `span` which also have the link in its `textContent`:

![image](https://github.com/odoo/odoo/assets/107679666/0a75e456-ee7a-4846-994c-244db7c4528e)

And in the Master fw-port you can completely remove the `input` :slightly_smiling_face: (and even probably the `span` around the `To join`)

Thanks and cheers!